devices/Ssd13xx/Ssd13xx.cs

```diff
@@ -527,5 +533,23 @@ public enum DisplayResolution
             0x00, // lower columns address =0
             0x10, // upper columns address =0
         };
+
+        /// <summary>
+        /// Reset display controller.
+        /// </summary>
+        /// <param name="rstPinNumber">Reset pin (some displays might be wired to share the microcontroller's
+        /// reset pin).</param>
+        private void Reset(int rstPinNumber)
+        {
+            var s_GpioController = new GpioController();            
```

Comment:
As pointed, this is not the correct pattern. Please use the proper pattern.
Also, you need to check that rstPinNumber is valid.